openstack/bms/v1/baremetalservers/results.go

```diff
@@ -139,3 +139,9 @@ func (r cloudServerResult) Extract() (*CloudServer, error) {
 	err := r.ExtractInto(&s)
 	return s.Server, err
 }
+
+func (r JobResult) Extract() (*JobStatus, error) {
```

Comment:
1. you should define this method in results_job.go;
2. JobResult already impl this method which named `ExtractJobStatus`, please check it.